extensions/controllers/sandboxclaim_controller.go

```diff
@@ -132,6 +190,17 @@ func (r *SandboxClaimReconciler) computeReadyCondition(claim *extensionsv1alpha1
 	}
 
 	// Sandbox should be non-nil if err is nil
+	if isExpired {
+		return metav1.Condition{
+			Type:               string(sandboxv1alpha1.SandboxConditionReady),
+			Status:             metav1.ConditionFalse,
+			Reason:             ClaimExpiredReason,
```

Comment:
Nit: Suggest removing `(DeletionPolicy=Retain)` from the status message. This decouples the status message from the specific configuration, as suggested in previous reviews.